### PR TITLE
Header Section

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
Created the Portfolio: setup and mobile first project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav><img class="union" src="/images/Union.png"  alt=""></nav>
+    </header>
+    <section class="introduction">
+        <h1>I’m Isabella
+            Glad to see you!</h1>
+        <h2>I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h2>
```
Comment (zieeco) [inline index.html]: - Hey, kindly make this a p tag as this is a paragraph and the `p` tag is used semantically to denote a paragraph.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <h3>LET'S CONNECT</h3></a>
+        <img src="/images/social media icons.png" alt="">
```
Comment (zieeco) [inline index.html]: - Hey, this part of the social media icons should be in `ul` container, which encloses each `li` as the list items, containing an `a` tag which wraps each `img` making the clickable and referencing users to the links of the different social media icons link. Here's what i mean:
```
    <ul class="all-list">
        <li><a href=""><img src="" alt=""></a></li>
        <li><a href=""><img src="" alt=""></a></li>
         .... and the remaining ....  
      </ul>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav><img class="union" src="/images/Union.png"  alt=""></nav>
```
Comment (zieeco) [inline index.html]: - The hamburger icon is not showing up. please, heck the image path to correct this issue.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    </header>
+    <section class="introduction">
```
Comment (zieeco) [inline index.html:18]: - Hi, please, the section is missing the background image. You can check the Figma design to see the image to add it in your design to make your design look close to the Figma design.

File: README.md [hunk 5]
```diff
@@ -273,15 +257,15 @@ I would like to thank...
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./LICENSE)licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (zieeco) [inline README.md]: - Please, let's kindly remove this placeholder text which servers as instruction telling you how important it is to have a license in your repositories. They're not expected to be in your readme after making an update to the template.

File: README.md [hunk 6]
```diff
@@ -273,15 +257,15 @@ I would like to thank...
...
+  - [Answer_2] -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./LICENSE)licensed.
```
Comment (zieeco) [inline README.md]: - Hey, the license link is currently broken. please, let's fix this issue to make the readme more professional.

File: README.md [hunk 7]
```diff
@@ -205,29 +193,27 @@ Example:
...
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[I will add other sections]**
+
```
Comment (zieeco) [inline README.md]: - Please, kindly make this more descriptive by telling users what section you want to add. e.g I will add the `Contact section`, `Project section`, `Desktop version of the design`. etc. This is more descriptive.

File: README.md [hunk 8]
```diff
@@ -92,48 +90,38 @@ After you're finished please remove all the comments and instructions!
...
 > Add a link to your deployed project.
 
 - [Live Demo Link](https://yourdeployedapplicationlink.com)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```
Comment (zieeco) [inline README.md:118]: - Hey, let's improve this section by adding some descriptive instructions on how the users can have your project run locally on their machine to make the readme more professional.
- Please, kindly remove the test section as you won't be adding any test on this project. see the image below to update them:

Screenshot from 2023-02-21 13-05-11

File: README.md [hunk 9]
```diff
@@ -205,29 +193,27 @@ Example:
 
 > Mention all of the collaborators of this project.
```
Comment (zieeco) [inline README.md]: - Please, let's also remove this placeholder text as this is here to guide you on how to add content to the readme as you should remove them after making the necessary changes.

File: README.md [hunk 10]
```diff
@@ -273,15 +257,15 @@ I would like to thank...
 
 - **[Question_2]**
 
-  - [Answer_2]
+  - [Answer_2] -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (zieeco) [inline README.md:201]: - Hey, please, let's remove this link as its currently duplicated.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Bellagirl-maker ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Bellagirl-maker) [reply index.html]: @zieeco the hamburger icon is on the page.  Could you please check again

File: README.md (hunk 6 above)
Comment (Bellagirl-maker) [reply README.md]: @zieeco I have added a link to the license

File: README.md (hunk 7 above)
Comment (Bellagirl-maker) [reply README.md]: @zieeco I have change the description here to " I will add Projects sections"

File: README.md (hunk 8 above)
Comment (Bellagirl-maker) [reply README.md:118]: @zieeco I removed the run test part of the Getting started

File: README.md (hunk 10 above)
Comment (Bellagirl-maker) [reply README.md:201]: @zieeco I  have removed this duplicated part

File: index.html (hunk 1 above)
Comment (Bellagirl-maker) [reply index.html]: @zieeco This part is done

File: index.html (hunk 2 above)
Comment (Bellagirl-maker) [reply index.html]: @zieeco done as well

File: index.html (hunk 4 above)
Comment (Bellagirl-maker) [reply index.html:18]: @zieeco I have added the background image

Comment (souad988) [reply index.html:18]: @Bellagirl-maker  well done you made the requested change 💪, but please as requested by previous reviewer check figma design and try your best to replicate it, you added the background image 👍 but the background color is different 
**figma design:**
figma temp1

**your design:**
student temp1

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Bellagirl-maker , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Setup linters correctly and pass all check.
✔Good communication with reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fix the alignment of your body as it has a margin , and also your container should be centered, kindly make sure to stick to figma design.
student temp1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @Bellagirl-maker 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav><img class="union" src="./images/Union.png"  alt=""></nav>
+    </header>
+    <section class="introduction">
```
Comment (joel-kalema) [inline index.html:18]: - [ ] [OPTIONAL] Good job fixing the background image 😃 , Kindly you may fix the margins to look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it was [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |mobile01| |mobile | |

**Pull request merged**